python/cac_tripplanner/templates/home.html

```diff
@@ -62,8 +62,35 @@
     <div class="places">
         <div class="places-content">
             <header class="places-header">
-                <h1>Places we love</h1>
-                <a href="#" class="map-view-btn">Map View</a>
+                <div class="places-header-content">
```

Comment:
Is `div.places-header-content` needed? Looks like it's the only child of `header.places-header`.

Reply:
Added for styling reasons. The outer `div` forces its margins out to fill the background color to light grey, as in the Marvel prototype. The inner `div` has margins to match the content sizing.

This was the best I could come up with, as Marvel is a black box that cannot be inspected to see elements or styling in use, so there's no styling or functional prototype to go off of. A cleanup PR for styling would be welcome.

Reply:
Got it. No problem.